test/multilocale/numLocales/bradc/testVFlag.launcher-aprun.goodstart

```diff
@@ -1 +1 @@
-aprun  -q -cc none -dN -n1 -N1 -j0 ./testVFlag_real -v -nl 1 EXECOPTS
```

Comment:
Why are there no EXECOPTS for aprun anymore?

Reply:
Hmm, that's odd; let me check that.